### ISSUE TITLE
`Warning: * is used but has no driver.` better be recognized as a Warning in the pre-synthesis checks

### ISSUE THREAD

**Issue by mo-hosni** [efabless/openlane2#387]
### Description

Currently, those are recognized as errors in both pre-synthesis and post-synthesis checks.
I think it better be a Warning in pre-synthesis and an error in post-synthesis as the signals without drivers are usually tied low in synthesis.
```
Warning: Wire user_project_wrapper.\io_out [4] is used but has no driver.
Warning: Wire user_project_wrapper.\io_out [3] is used but has no driver.
Warning: Wire user_project_wrapper.\io_out [2] is used but has no driver.
```

**Comment by donn** [comment]
With regards to this -- the problem is we can't tell what's intentionally left undriven and what's not would it not be more prudent to ask users to explicitly tie them low?

**Comment by donn** [comment]
Resolved in #461.
